web3/providers/websocket/websocket_v2.py

```diff
@@ -67,6 +67,7 @@ def __init__(
         endpoint_uri: Optional[Union[URI, str]] = None,
         websocket_kwargs: Optional[Dict[str, Any]] = None,
         request_timeout: Optional[float] = DEFAULT_PERSISTENT_CONNECTION_TIMEOUT,
```

Comment:
If I want to define a different `subscription_response_queue_size` to be passed on to the `PersistentConnectionProvider`, should it not be available as an optional arg here, to then be passed on via `super().__init__`? Or is there a different place I'm missing?

Reply:
Good catch, We can probably leave default as `None` here and let the superclass set the default value too. I'll put that config in 👍🏼